pedometer/chapter.md

```diff
@@ -212,22 +216,22 @@ In accounting for these four scenarios, we've managed to bring our messy $a(t)$
 The problem, at first glance, looked straightforward. However, the real world and real people threw a few curve balls our way. Let's recap how we solved the problem:
 
 1. We started with total acceleration, $(x(t), y(t), z(t))$.
-2. We used a low-pass filter to split total acceleration into user acceleration, $(x_{u}(t), y_{u}(t), z_{u}(t))$, and gravitational acceleration, $(x_{g}(t), y_{g}(t), z_{g}(t))$.
+2. We used a low-pass filter to split total acceleration into user acceleration $(x_{u}(t), y_{u}(t), z_{u}(t))$ and gravitational acceleration $(x_{g}(t), y_{g}(t), z_{g}(t))$.
 3. We took the dot product of $(x_{u}(t), y_{u}(t), z_{u}(t))$ and $(x_{g}(t), y_{g}(t), z_{g}(t))$ to obtain the user acceleration in the direction of gravity, $a(t)$.
 4. We used a low-pass filter again to remove the high-frequency component of $a(t)$, removing noise.
 5. We used a high-pass filter to cancel the low-frequency component of $a(t)$, removing slow peaks.
 6. We set a threshold to ignore short peaks.
 7. We used hysteresis to avoid double-counting steps with bumpy peaks.
 
-As software developers in a training or academic setting, we may have been presented with a perfect signal and asked to write code to count the steps in that signal. While that may have been an interesting coding challenge, it wouldn't have been something we could apply in a live situation. We saw that in a reality with gravity and people thrown into the mix, the problem was a little more complex. We used mathematical tools to address the complexities, and were able to solve a real-world problem. It's time to translate our solution into code.
+As software developers in a training or academic setting, we may have been presented with a perfect signal and asked to write code to count the steps in that signal. While that may have been an interesting coding challenge, it wouldn't have been something we could apply in a live situation. We saw that in reality, with gravity and people thrown into the mix, the problem was a little more complex. We used mathematical tools to address the complexities, and were able to solve a real-world problem. It's time to translate our solution into code.
 
 # Diving Into Code
 
-Our goal for this chapter is to create a web application in Ruby that accepts accelerometer data, parses, processes, and analyzes it, and returns the number of steps taken, the distance traveled, and the elapsed time.
+Our goal for this chapter is to create a web application in Ruby that accepts accelerometer data, parses, processes, and analyzes it, and returns the number of steps taken, the distance travelled, and the elapsed time.
```

Comment:
Changed this to "British" spelling since later in the chapter you use "labour". Let me know if you would prefer to consistently use US spelling throughout the chapter.